source/vibe/data/json.d

```diff
@@ -69,14 +69,23 @@ import std.traits;
 	a JSONException. Additionally, an explicit cast or using get!() or to!() is
 	required to convert a JSON value to the corresponding static D type.
 */
+
 struct Json {
 	private {
 		// putting all fields in a union results in many false pointers leading to
 		// memory leaks and, worse, std.algorithm.swap triggering an assertion
 		// because of internal pointers. This crude workaround seems to fix
 		// the issues.
-		void*[2] m_data;
+		void*[max((BigInt.sizeof+(void*).sizeof-1)/(void*).sizeof, 2)] m_data;
 		ref inout(T) getDataAs(T)() inout { static assert(T.sizeof <= m_data.sizeof); return *cast(inout(T)*)m_data.ptr; }
+
+		void initBigInt()
+		{
+			BigInt init_;
+			(cast(ubyte*)m_data.ptr)[0 .. BigInt.sizeof] = (cast(ubyte*)&init_)[0 .. BigInt.sizeof];
```

Comment:
I think it is not a bug. Simply dmd and gdc uses another BigInt implementation: with inline asm.
The general idea: BigInt.init contains a BigUInt instance, which initially should contatin a [0] array.
When we get BigInt via `getDataAs`, it may contain the garbarge (null or something worse) (and BigUInt invariant is fault). Thus, we should explicitly write BigInt.init to storage before the first using.